print_env.sh

```diff
@@ -0,0 +1,49 @@
+# Reports relevant environment information useful for diagnosing and
+# debugging cuDF issues.
+# Usage: 
+# "./print_env.sh" - prints to stdout
+# "./print_env.sh > env.txt" - prints to file "env.txt"
+
+echo "***git log --decorate -n 1***"
+git log --decorate -n 1
+echo " "
+
+echo "***cat /etc/*-release***"
+cat /etc/*-release
+echo " "
+
+echo "***uname -a***"
+uname -a
+echo " "
+
+echo "***nvidia-smi***"
+nvidia-smi
+echo " "
+
+echo '***conda list "pandas|arrow|numpy|^python$"***'
+conda list 'pandas|arrow|numpy|^python$'
+echo " "
+
+echo "***which cmake && cmake --version***"
+which cmake && cmake --version | grep "version"
+echo " "
+
+echo "***which g++ && g++ --version***"
+which g++ && g++ --version | grep "g++"
+echo " "
+
+echo "***which nvcc && nvcc --version***"
+which nvcc && nvcc --version | grep "release"
+echo " "
+
+echo "***echo NUMBAPRO_NVVM***"
+echo $NUMBAPRO_NVVM
```

Comment:
May also be useful to check `LD_LIBRARY_PATH` and `PATH` as well

Reply:
Agree, and CONDA_PREFIX. If there is a better way to detect that they are in a conda environment, that would be good to use.

Reply:
Also PYTHONPATH